lib/cmds/feeling_lucky.py

```diff
@@ -18,11 +18,12 @@ def __init__(self, bot):
     @commands.command(name=CMD)
     async def feeling_lucky(self, ctx):
         rand = random.randint(1, 3)
+        message = None
 
         match rand:
             # Item Event - Add an item to the player's inventory
             case 1:
-                item_name, quantity = self.__give_item(ctx.author.id)
+                item_name, quantity = await self.__give_item(ctx.author.id)
```

Comment:
noticed a bug here myself. Will update commit sooon

Reply:
I will commit this